src/main/avro/lessonStatusItem.avsc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Just curious, does the Avro format allow for comments?  I can comments being very helpful to the reader.